### PR TITLE
Releasing charts automatically

### PR DESCRIPTION
**Pull request by srashee1**
Most of this PR is moving all the charts to a directory called charts.
Adding a github action to automatically package and release the charts (Prerequisite: Change the hosted github page to host from a new branch "gh-pages" from the "/" directory after merging)
Organizing the charts alphabetically and adding local references for automating urbanos chart release
Deleting outdated platform charts

/urban-os-deploy#266

co-authored-by: Kyle Smith <kyle.r.smith@accenture.com>

### CODE REVIEW

**Comment by christinepoydence** [comment]
I don't see the charts directory that all the charts were moved to in this PR. Is there another repo with that code?

**Comment by srashee1** [comment]
> I don't see the charts directory that all the charts were moved to in this PR. Is there another repo with that code?

Nope, good catch, they were being git ignored. They're all in the charts/ repo now

File: charts/urban-os/Chart.yaml [hunk 1]
```diff
@@ -5,63 +5,63 @@ name: urban-os
...
-  - name: valkyrie
-    version: 2.6.1
-    repository: https://Datastillery.github.io/charts
-    condition: valkyrie.enabled
   - name: andi
-    version: 2.1.0
-    repository: https://Datastillery.github.io/charts
-    condition: andi.enabled
+    version: 2.1.3
+    repository: file://../andi
```
Comment (narruda789) [inline charts/urban-os/Chart.yaml:10]: How are these local references going to affect the way a future developer will use the urban-os chart?

**Review by narruda789 — APPROVED** [review]
Looks great! Left one comment, but it may not be a high priority.

File: charts/urban-os/Chart.yaml (hunk 1 above)
Comment (srashee1) [reply charts/urban-os/Chart.yaml:10]: The urbanOS chart will by default now point to the local charts in this repository so any changes in those repositories will be picked up when building the urbanos chart. This is also how we're going to turn this into a 1 PR process

Comment (narruda789) [reply charts/urban-os/Chart.yaml:10]: Sounds good!

**Pull request merged**